make-release.sh

```diff
@@ -33,55 +76,114 @@ bump_version () {
 
   echo "Updating project version to ${NEXT_VERSION}"
   # change version/version.go file
-  sed -i version/version.go -r -e "s#(Version = \")(v?[0-9.]+)(\")#\1${NEXT_VERSION}\3#g"
+  sed -i version/version.go -r -e "s#(Version = \")(v?[0-9.]+)(\")#\1v${NEXT_VERSION}+rc\3#g"
+  sed -i deploy/templates/components/csv/clusterserviceversion.yaml -r -e "s#(name: devworkspace-operator.)(v[0-9.]+)#\1v${NEXT_VERSION}#g"
+  sed -i deploy/templates/components/csv/clusterserviceversion.yaml -r -e "s#(version: )([0-9.]+)#\1${NEXT_VERSION}#g" 
   if [[ ! -z $(git status -s) ]]; then # dirty
-    git add version/version.go
     COMMIT_MSG="[release] Bump to ${NEXT_VERSION} in ${BUMP_BRANCH}"
     git commit -asm "${COMMIT_MSG}"
   fi
   git pull origin "${BUMP_BRANCH}"
 
   set +e
-  PUSH_TRY="$(git push origin "${BUMP_BRANCH}")"
+  [[ -v "$DRY_RUN" ]] && PUSH_TRY="$(git push origin "${BUMP_BRANCH}")" || PUSH_TRY="protected branch hook declined"
```

Comment:
I don't know if `-v` is the right option here; also it seems to have been added in bash `4.2` (news to me) so we'll probably run into issues when someone with a mac tries to run it :D

Reply:
good catch. I'll replace it with -z.

Reply:
Well, shell check suggests -n instead https://github.com/koalaman/shellcheck/wiki/SC2236.
but now I rewrote it in the following way:
```bash
  [[ -z "$DRY_RUN" ]] && echo "dry run command" \
    || echo "real command"
```

Reply:
then shellcheck suggest not to use such construction for our purposes https://github.com/koalaman/shellcheck/wiki/SC2015 )